src/Speech/SpeechClient.php

```diff
@@ -188,6 +188,13 @@ public function __construct(array $config = [])
      *           to favor specific words and phrases in the results. Please see
      *           [SpeechContext](https://cloud.google.com/speech/reference/rest/v1/RecognitionConfig#SpeechContext)
      *           for more information.
+     *     @type bool $enableWordTimeOffsets If set to `true`, a list of
+     *           `word_time_offsets` are returned with the top alternative. If
```

Comment:
A few quick notes, otherwise LGTM!

- `word_time_offsets` came back camel case in my testing
- Do you think it is worth adding a note to [the result class](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/src/Speech/Result.php#L71) regarding the new offsets that may be returned?